### DISCUSSION TITLE
建议加强字字幕功能

### DISCUSSION THREAD

**Discussion by Re-hd** [ywmoyue/biliuwp-lite#254] [General]
能不能像B站官方一样设计一个智能字幕？

**Comment by ywmoyue** [comment]
什么意思

**Comment by Re-hd** [comment]
就是能不能在所有视频都加一个智能字幕，否则有些视频本身没有字幕，在公共场合不能外放，就挺难受的。

**Reply by ywmoyue** [reply]
b站也不是每个视频都有ai字幕，b站的ai字幕接口也已经接入bililite使用了。
后续有考虑对接ai字幕服务（需要自行部署且只能给离线缓存的视频用，否则对性能消耗太大也太慢了

**Reply by Re-hd** [reply]
我看windows自己的实时字幕选项性能消耗还好啊

**Reply by ywmoyue** [reply]
试了下Win自带实时字幕确实不错，我看看吧

**Reply by Re-hd** [reply]
OK，感谢
